compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5265,6 +5274,9 @@ THRIFT_REGISTER_GENERATOR(
     "    android_legacy:  Do not use java.io.IOException(throwable) (available for Android 2.3 and "
     "above).\n"
     "    option_type:     Wrap optional fields in an Option type.\n"
+    "    handle_runtime_exceptions:\n"
+    "                     Generated services will handle RuntimeException as "
```

Comment:
The output of `thrift --help` is already quite long, so I don't think an extra line or two is going to matter. What probably matters more is brevity and formatting for readability.

How about simply: `Send TApplicationException to clients when RuntimeException occurs on the server` ?